### PR TITLE
Fixes for planning with tolerances

### LINKED ISSUE #2: Tolerances not correctly passed in trajopt planner
There are some places where the waypoint is converted to a VectorXd and tolerances get lost. I have fixed it in this branch, but I need to copy it to this repo

https://github.com/mpowelson/tesseract/tree/updated_dev

### PR DESCRIPTION
**Pull request by mpowelson**
Closes #2

### CODE REVIEW

File: tesseract_motion_planners/include/tesseract_motion_planners/trajopt_ifopt/problem_generators/default_problem_generator.h [hunk 1]
```diff
@@ -37,10 +37,10 @@ TESSERACT_COMMON_IGNORE_WARNINGS_POP
 namespace tesseract_planning
 {
 std::shared_ptr<TrajOptIfoptProblem>
-DefaultTrajoptProblemGenerator(const std::string& name,
-                               const PlannerRequest& request,
-                               const TrajOptIfoptPlanProfileMap& plan_profiles,
-                               const TrajOptIfoptCompositeProfileMap& composite_profiles);
+DefaultTrajoptIfoptProblemGenerator(const std::string& name,
```
Comment (Levi-Armstrong) [inline tesseract_motion_planners/include/tesseract_motion_planners/trajopt_ifopt/problem_generators/default_problem_generator.h:40]: Should TrajOpt have a capital O?

File: tesseract_motion_planners/include/tesseract_motion_planners/trajopt_ifopt/profile/trajopt_ifopt_default_composite_profile.h [hunk 2]
```diff
@@ -46,19 +46,21 @@ class TrajOptIfoptDefaultCompositeProfile : public TrajOptIfoptCompositeProfile
...
   /** @brief This default to all ones, but allows you to weight different joints */
   Eigen::VectorXd velocity_coeff;
   /** @brief If true, a joint acceleration cost with a target of 0 will be applied for all timesteps Default: false*/
-  bool smooth_accelerations = true;
+  bool smooth_accelerations = false;
   /** @brief This default to all ones, but allows you to weight different joints */
   Eigen::VectorXd acceleration_coeff;
   /** @brief If true, a joint jerk cost with a target of 0 will be applied for all timesteps Default: false*/
-  bool smooth_jerks = true;
+  bool smooth_jerks = false;
```
Comment (Levi-Armstrong) [inline tesseract_motion_planners/include/tesseract_motion_planners/trajopt_ifopt/profile/trajopt_ifopt_default_composite_profile.h:63]: Why a default to false?

File: tesseract_motion_planners/include/tesseract_motion_planners/trajopt_ifopt/problem_generators/default_problem_generator.h (hunk 1 above)
Comment (mpowelson) [reply tesseract_motion_planners/include/tesseract_motion_planners/trajopt_ifopt/problem_generators/default_problem_generator.h:40]: Yeah it should.

File: tesseract_motion_planners/include/tesseract_motion_planners/trajopt_ifopt/profile/trajopt_ifopt_default_composite_profile.h (hunk 2 above)
Comment (mpowelson) [reply tesseract_motion_planners/include/tesseract_motion_planners/trajopt_ifopt/profile/trajopt_ifopt_default_composite_profile.h:63]: These haven't been implemented in trajopt_ifopt yet. It won't cause any problems to have them as true, but you'll get warnings every time.

Comment (Levi-Armstrong) [reply tesseract_motion_planners/include/tesseract_motion_planners/trajopt_ifopt/profile/trajopt_ifopt_default_composite_profile.h:63]: Ok, that makes sense.

**Comment by Levi-Armstrong** [comment]
If you rebase on the latest CI should run now.

**Review by Levi-Armstrong — APPROVED** [review]

**Pull request merged**